### ISSUE TITLE
XY Chart With Zoom Disabled Still Zooms When Scrolling Quickly

### ISSUE THREAD

**Issue by keegandonley** [amcharts/amcharts5#303]
I'm running into a problem using the latest release (`5.1.5`) with panY. 
My chart configuration looks like this:
```js
{
        panY: true,
        wheelY: 'panY',
        panX: false,
        wheelStep: 2
}
```
However as you can see demonstrated in the video below, when using the wheel to pan up and down, the chart is still changing zoom levels:

https://user-images.githubusercontent.com/12700819/155192245-285b16aa-7d32-4159-9446-4a442322a3eb.mov

It always snaps back to being correct, but does not look good while the pan is happening

**Comment by zeroin** [comment]
Could I ask you to create a codepen of exactly the same chart? It would help me a lot to understand what's happening. And there are no `zoomX` and `zoomY` settings on a chart, so they do not do anything.

**Comment by keegandonley** [comment]
I can work on creating a reproduction when I have more bandwidth. For now it's just something I have to live with until I have more time.

**Comment by keegandonley** [comment]
@zeroin I've got a reproduction up on Codepen here https://codepen.io/keegandonley/pen/RwjOybw. Apologies for the complexity, but this is an exact reproduction of the chart that's exhibiting the weird scrolling behavior. I've confirmed it's present in this codepen for me.

**Comment by martynasma** [comment]
Fixed in [5.1.9](https://github.com/amcharts/amcharts5/releases/tag/5.1.9).

## [5.1.9] - 2022-03-17

### Fixed
- Fixed the issue with resizing a chart inside of a flexbox. ([Issue 311](https://github.com/amcharts/amcharts5/issues/311))
- Fixed jumpy axis when panning an `XYChart` with mouse wheel.
- Panning XYChart with mouse wheel had incorrect direction if axis was inversed.
- If axis range had a gradient fill, gradient used to spread only through visible area, making it look icorrect.
- After pinch zooming and `XYChart` or zooming the chart with no animated theme used, CPU usage used to increase.
- `DateAxis` with `baseIntervals` `timeUnit` set to `"month"` could show incorrect month in the axis tooltip.
- If a `Graphics` had a gradient `fill`/`stroke` and initially was invisible, setting it to visible did not make the gradient to appear.
- While zooming `LineSeries`, a part of the line to the left of the first visible data point could disappear.

[Full change log](https://github.com/amcharts/amcharts5/blob/master/packages/shared/CHANGELOG.md).

[Download options](https://www.amcharts.com/download/).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.

**Comment by keegandonley** [comment]
Looks great thank you!
